### PR TITLE
Boxes (and sterile mask)

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Just more bloat content. I'm not making empty variants for these boxes like the crates or lockers.

Added 
- donkpocket box 
- buncha boxes for stuff
- sterile mask

I made sure to organize things neatly yada yada.

I'm gonna tie this into #1835 so this will be completed before that.

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
Added "filled" versions of the backpacks for spawning start gear.

File: Resources/Textures/Clothing/Masks/mask_sterile.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from https://github.com/tgstation",
```
Comment (DrSmugleaf) [inline Resources/Textures/Clothing/Masks/mask_sterile.rsi/meta.json:8]: Missing a commit id here.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
LGTM except for one rsi's attribution missing a commit id.

**Comment by SweptWasTaken** [comment]
Ready

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**